.github/workflows/assignIssue.yml

```diff
@@ -55,8 +55,8 @@ jobs:
           echo ${{steps.week.outputs.odd}}
           echo ${{steps.hour.outputs.hour}}
         shell: bash
-      - name: Even late friday (David)
-        if: steps.proceed.outputs.result == 1 && steps.week.outputs.odd == 0 && steps.day.outputs.number == 5 && steps.hour.outputs.hour >= 16
+      - name: Saturday (Claudia)
```

Comment:
@claudiaregio offered to take Saturdays off of me :)
So the new order is:
- Sunday & Monday: me
- Tuesday: Ian
- Wednesday: Rich
- Thursday: Don
- Friday & Saturday: Claudia